assets/src/edit-story/components/canvas/useCanvasKeys.js

```diff
@@ -80,9 +82,10 @@ function useCanvasKeys(ref) {
       const dirX = getArrowDir(key, 'ArrowRight', 'ArrowLeft');
       const dirY = getArrowDir(key, 'ArrowDown', 'ArrowUp');
       const delta = shiftKey ? 1 : MOVE_COARSE_STEP;
+      const multiplier = isRTL ? -1 : 1;
```

Comment:
Hmm. Even though it's RTL editor, it's not a guarantee that this should be a RTL document. And even if it was, x/y/etc don't have to change. I think we need to separate editor/document as much as possible for this.